docs/source/install.rst

```diff
@@ -143,7 +141,7 @@ These optional dependencies and their minimum supported versions are listed belo
 +------------------+-----------------+---------------------------------------------------------------------------------------------------------+
 | `psutil`_        | ``>=5.7.2``     | Factor CPU affinity into CPU count, intelligently infer blocksize when reading CSV files                |
 +------------------+-----------------+---------------------------------------------------------------------------------------------------------+
-| `pyarrow`_       | ``>=7.0``       | Support for Apache Arrow datatypes & engine when storing/reading Apache ORC or Parquet files            |
+| `pyarrow`_       | ``>=7.0``       | Support for Apache Arrow datatypes, `tdigest` & engine when storing/reading Apache ORC or Parquet files |
```

Comment:
tdigest needs double backticks for sphinx rendering

Reply:
Done, but needed to extend the table a bit.https://github.com/dask/dask/pull/10611/commits/97275dcd9df24bc550d1f3b96473678fa17a3c0d